ui/app/templates/components/transit-form-show.hbs

```diff
@@ -63,7 +63,7 @@
         />
       {{/if}}
       {{#if (eq @mode "show")}}
-        {{#if (or @capabilities.canUpdate @capabilities.canDelete)}}
+        {{#if (or @model.canUpdate @model.canDelete)}}
```

Comment:
should this be `@model` or `@key`? Sorry - hard to follow all the capabilities and where they trace back to!

Reply:
100% I went back and forth and I'll double check just to be sure. the edit form uses key as the model, and while this could also use key, the param model is the standard.

Reply:
[Here](https://github.com/hashicorp/vault/pull/28563/files#diff-d0e181623443a26de2e79ee21726bd42287d5ce9c625ce6bda10e8f68e596bc7R50) the model was passed in for the show form.